pom.xml

```
@@ -55,6 +55,7 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
+            <scope>provided</scope>
```

```suggestion
            <scope>provided</scope>
```

This must be removed, otherwise it cannot be compiled automatically.

---

Done